### PR TITLE
Improve macOS messages

### PR DESCRIPTION
**Pull request by mhsmith**
Some simplifications based on my experience of doing the tutorial last month on a freshly-installed machine.

Related: https://github.com/beeware/beeware/pull/191

## PR Checklist:

- [x] All new features have been tested
- [x] All new features have been documented
- [x] I have read the **CONTRIBUTING.md** file
- [x] I will abide by the code of conduct

### CODE REVIEW

File: src/briefcase/integrations/xcode.py [hunk 1]
```diff
@@ -87,13 +87,10 @@ def ensure_command_line_tools_are_installed(tools: ToolCache):
...
-Xcode command line developer tools are not installed.
+The command line developer tools are not installed.
 
-You should be shown a dialog prompting you to install Xcode and the
-command line tools. Select "Install" to install the command line developer
-tools.
-
-Re-run Briefcase once that installation is complete.
+You should be shown a dialog prompting you to install them. Select "Install"
+to continue, and re-run Briefcase once that installation is complete.
```
Comment (mhsmith) [inline src/briefcase/integrations/xcode.py:93]: Clarify the distinction between Xcode and the command line developer tools.

File: src/briefcase/integrations/xcode.py [hunk 2]
```diff
@@ -227,21 +224,19 @@ def ensure_xcode_is_installed(
...
             else:
                 preamble = """\
 You have the Xcode command line tools installed; however, Briefcase requires
-a full Xcode install. Xcode can be downloaded from the macOS App Store.
-
-Once you have installed Xcode, you can make it the active developer directory
-by running:
+a full Xcode install. Xcode can be downloaded from the macOS App Store at
+<https://apps.apple.com/au/app/xcode/id497799835?mt=12>.
 """
```
Comment (mhsmith) [inline src/briefcase/integrations/xcode.py:235]: After installing Xcode, it becomes the active developer directory automatically, so the `xcode-select` command is not required in this case.

File: src/briefcase/integrations/xcode.py [hunk 3]
```diff
@@ -284,14 +279,12 @@ def confirm_xcode_license_accepted(tools: ToolCache):
...
+can run this command and accept the license when prompted:
 
     $ sudo xcodebuild -license
 
-at the command line and accepting the license there.
-
-Briefcase will try the command line version of this command now. You will need
-to enter your password (Briefcase will not store this password anywhere).
+Briefcase will try to run this command now. You will need to enter your
+password (Briefcase will not store this password anywhere).
```
Comment (mhsmith) [inline src/briefcase/integrations/xcode.py:287]: Nothing substantive here, just wording improvements.

**Review by freakboy3742 — APPROVED** [review]
These all make sense to me 👍

**Pull request merged**